react/components/Header/Summary.tsx

```diff
@@ -29,78 +28,79 @@ const OrderSummary: FunctionComponent<Props & InjectedIntlProps> = ({
   const longestPickUpEstimate = estimateCalculator.getLatestSla(totalPickUps)
 
   return (
-    <section className="w-80 center bb b--muted-4 pb8">
-      <PageBlock variation="half">
-        <article className="center">
-          <p className="t-heading-4 tc bb b--muted-5 mv0 pb5 c-on-base">
-            {intl.formatMessage({ id: 'summary.shipping' })}
-          </p>
-          <p className="mb0 pt5 t-body mt2">
-            <strong>
-              {intl.formatMessage(
-                { id: 'summary.items' },
-                {
-                  itemsQuantity: deliveryItemsQuantity,
-                }
-              )}
-            </strong>
+    <section
+      data-testid="summary"
+      className="w-90 w-80-ns center bb b--muted-4 pb8 flex justify-between flex-wrap flex-nowrap-m"
+    >
+      <article className="ba b--muted-5 bw1 br3 w-50-m w-100 mr4-m mb4 mb0-m">
+        <p className="t-heading-5 tc bb b--muted-5 bw1 pb5 c-on-base">
+          {intl.formatMessage({ id: 'summary.shipping' })}
+        </p>
+        <p className="mb0 t-body ph6 mt6">
+          <strong>
             {intl.formatMessage(
-              { id: 'summary.shipping.quantity' },
+              { id: 'summary.items' },
               {
-                shippings: totalDeliveries.length,
+                itemsQuantity: deliveryItemsQuantity,
               }
             )}
-          </p>
-          <p className="t-heading-4 mt4">
-            <TranslateEstimate
-              shippingEstimate={longestDeliveryEstimate.shippingEstimate}
-            />
-          </p>
-          <p className="c-muted-2 self-baseline mb0">
+          </strong>
+          {intl.formatMessage(
```

Comment:
i know this is not related directly to this PR, but why this isn't `<FormattedMessage/>` being used here?

Reply:
Just to maintain a pattern, I try not to use `<FormattedMessage />` whenever I can because of the extra `<span>` that is wrapped around the text. As I'm not passing HTML elements down to the message, I didn't see a reason to use `<FormattedMessage />`.
Is it cleaner in some way? Maybe to remove the wrapping `<p>` tags?